app/controllers/new_gestionnaire/concerns/create_avis_concern.rb

```diff
@@ -0,0 +1,30 @@
+module NewGestionnaire
+  module Concerns
+    module CreateAvisConcern
+      extend ActiveSupport::Concern
+
+      private
+
+      def create_avis_from_params(dossier, confidentiel = false)
+        confidentiel ||= create_avis_params[:confidentiel]
+        avis = Avis.new(create_avis_params.merge(claimant: current_gestionnaire, dossier: dossier, confidentiel: confidentiel))
+
+        if avis.save
+          flash.notice = "Une demande d'avis a été envoyée à #{avis.email_to_display}"
```

Comment:
Je ne sais pas quoi penser du fait que le concern définisse des flash, j'hésite entre "je m'attendrais à ce que le concern ne fasse que des trucs back" et "why not, après tout"

Reply:
Oui. Mon point de vue là dessus : ça n’aurait pas du tout sa place dans une service métier, là avec le concern j’ai plutôt voulu faire quelque chose dans l’esprit d’un partial ou d’un helper. Juste un petit bout de contrôleur réutilisable.